sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/TableRowToStorageApiProtoTest.java

```diff
@@ -1671,6 +1672,44 @@ public void testIgnoreUnknownRepeatedNestedField() throws Exception {
     assertEquals("foobar_doubly_nested", unknownDoublyNestedStruct.get("unknown_doubly_nested"));
   }
 
+  @Test
+  public void testIgnoreUnknownRepeatedNestedField2() throws Exception {
```

Comment:
nit: more descriptive test name 
```suggestion
  public void testIgnoreUnknownRepeatedNestedFieldWithNoUnknowns() throws Exception {
```